src/mpid/ch3/channels/nemesis/netmod/ofi/ofi_init.c

```diff
@@ -183,7 +162,6 @@ int MPID_nem_ofi_init(MPIDI_PG_t * pg_p, int pg_rank, char **bc_val_p, int *val_
     /* --------------------------- */
     /* Free providers info         */
     /* --------------------------- */
-    MPL_free(provname);
```

Comment:
Let's not to change ch3 since the provider selection still depend on this CVAR. We can change the name to `MPIR_CVAR_CH3_OFI_USE_PROVIDER` though.

Reply:
I'd say if we're going to leave it, let's just leave it. As an added benefit, we can use the fact that the cvar is still there to print a warning in CH4...